### ISSUE TITLE
confirm exit from "create comment" and "create post" mode

### ISSUE THREAD

**Issue by firefoxlover** [dessalines/jerboa#1414]
Android has these annoying gestures you cant remove, and especially from the left edge I sometimes swipe back when I want to mark text for quoting.

When creating a post its similar.

Having the same "really want to exit?" dialog as in the "close app?" would be way more important.

**Comment by MV-GH** [comment]
> Android has these annoying gestures you cant remove, and especially from the left edge I sometimes swipe back when I want to mark text for quoting.

I think you can, you can choose between gestures or navigation bar

We had a draft system PR but it didn't get finished which instead of a exit confirmation it would show a popup, saying do you want to save this text? yes/no

**Comment by dessalines** [comment]
I like that the best. Maybe would should close this then as a dupe of #356

**Comment by firefoxlover** [comment]
> I think you can, you can choose between gestures or navigation bar

Yes but this is not an option. I mean removing just the left swipe.

**Comment by firefoxlover** [comment]
@dessalines are drafts even a thing currently?

Is there any implementation this could hook in?

(I again deleted a huge text accidentally...)

I think having a small confirmation is way easier?

**Comment by dessalines** [comment]
We don't have drafts yet, but I agree a confirm exist for these would be good to have until then.
